core/domain/exp_jobs_one_off.py

```diff
@@ -520,16 +520,19 @@ def map(item):
         if (item.states_schema_version <
                 AUDIO_DURATION_SECS_MIN_STATE_SCHEMA_VERSION):
             yield ('State schema version is not the '
-                   'minimum version expected', item.id)
+                   'minimum version expected, 31', 'EXP_ID: %s, '
+                   'STATE_SCHEMA_VERSION: %s' %
+                   (item.id, item.states_schema_version))
             return
 
         # Go through each exploration state to find voiceover recordings.
         # For batching them as commits per explorations.
         commit_cmds = []
-        states_changed_count = 0
-        unchanged_count = 0
-        failed_count = 0
+        exploration_states_changed_count = 0
+        exploration_states_unchanged_count = 0
+        exploration_states_failed_count = 0
         for state, state_value in item.states.items():
+            states_changed_count = 0
```

Comment:
Could you please clarify what this variable represents and how exactly does it differ from exploration_states_changed_count above? I think the name might be misleading but I'm a bit confused about what you actually want it to mean.

Also, do you need it to be a count? From what I can tell it's more a "state_is_changed = True/False" type of variable.

Reply:
Ok I will change it to that.

Reply:
ok